java/com/walmartlabs/logback/RiemannAppender.java

```diff
@@ -130,6 +129,8 @@ boolean isMinimumLevel(ILoggingEvent logEvent) {
     return logEvent.getLevel().isGreaterOrEqual(riemannLogLevel);
   }
 
+  protected final static int MAX_ATTEMPTS = 2;
```

Comment:
Not a big fan of declaring variables in the middle of the class. Maybe move this to the top like all the other instance/class variables?